<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Yeastar-PC" name="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension" tests="23" failures="2" timestamp="2 Nov 2018 02:36:47 GMT" time="937.146" errors="0">
  <testcase name="C2_callpark" time="53.189" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension"/>
  <testcase name="E2_cancelWhenBusy" time="40.319" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension"/>
  <testcase name="D1_callforward" time="32.833" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension"/>
  <testcase name="J_internal" time="48.455" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension"/>
  <testcase name="K2_Check_OneTouchRecord" time="4.883" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension"/>
  <testcase name="H1_resetToDefault" time="7.154" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension"/>
  <testcase name="B2_AttendTranferTimeout" time="44.603" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension">
    <failure type="java.lang.AssertionError" message="预计产生多条CDR，但没有找到匹配">
      <![CDATA[java.lang.AssertionError: 预计产生多条CDR，但没有找到匹配
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:348)
	at com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension.B2_AttendTranferTimeout(Extension.java:219)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- B2_AttendTranferTimeout -->
  <testcase name="K1_check_Voicemail" time="55.822" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension"/>
  <testcase name="A1_sip" time="43.297" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension">
    <failure type="java.lang.AssertionError" message=" 录音失败">
      <![CDATA[java.lang.AssertionError:  录音失败
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
	at com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension.A1_sip(Extension.java:136)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- A1_sip -->
  <testcase name="A3_fxs" time="35.355" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension"/>
  <testcase name="A2_sip" time="29.877" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension"/>
  <testcase name="A0_Register" time="29.093" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension"/>
  <testcase name="F2_NoAnswer" time="55.896" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension"/>
  <testcase name="C1_callpark" time="50.273" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension"/>
  <testcase name="I_Voicemail" time="79.825" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension"/>
  <testcase name="G1_dnd" time="15.300" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension"/>
  <testcase name="A4_fxs" time="40.353" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension"/>
  <testcase name="G2_canceldnd" time="30.994" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension"/>
  <testcase name="B3_BlindTransfer" time="40.836" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension"/>
  <testcase name="E1_whenbusy" time="48.859" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension"/>
  <testcase name="F1_NoAnswer" time="67.181" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension"/>
  <testcase name="D2_cancellForward" time="31.927" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension"/>
  <testcase name="B1_AttendTranfer" time="50.822" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension"/>
</testsuite> <!-- com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension -->
